### PR TITLE
bugfix: accept int in shell command

### PR DESCRIPTION
**Pull request by gruberrro**
`AttributeError: 'int' object has no attribute` when passing int in command

https://github.com/fabric/fabric/issues/1204

### CODE REVIEW

**Comment by ploxiln** [comment]
Looks good to me.

**Comment by gruber86** [comment]
@bitprophet @ploxiln Guys, do you know why the travis check failed? Is it related to my code?

File: tests/test_operations.py [hunk 1]
```diff
@@ -195,6 +195,18 @@ def test_prompt_with_default():
...
 
 
+def test_prefix_env_vars_non_strings():
+    """
+    _prefix_env_vars() test int, float, bool as env variable
+    """
+    values = [123, u'123', 1.11, True, False]
+    expect = ['123', '123', '1.11', 'True', 'False']
+    for index, value in enumerate(values):
+        env.shell_env = dict(VAR=value)
```
Comment (ploxiln) [inline tests/test_operations.py]: This `env.shell_env` is affecting other tests.
I think using the settings context manager might work: http://docs.fabfile.org/en/1.13/api/core/context_managers.html#fabric.context_managers.settings

**Comment by gruberrro** [comment]
@ploxiln @bitprophet ready to merge?

**Pull request closed without merging**